homeassistant/auth/providers/trusted_networks.py

```diff
@@ -46,7 +46,7 @@
                     [
                         vol.Or(
                             cv.uuid4_hex,
-                            vol.Schema({vol.Required(CONF_GROUP): cv.uuid4_hex}),
+                            vol.Schema({vol.Required(CONF_GROUP): str}),
```

Comment:
> The original test for the code passes because it did not test the CONFIG_SCHEMA. The config fixture is partially built and group information is injected at runtime, bypassing the schema validation.

Maybe you could adjust the tests accordingly, so the `CONFIG_SCHEMA` gets tested?

Reply:
Tests added.